modules/gdscript/gdscript_editor.cpp

```diff
@@ -671,7 +671,12 @@ static String _make_arguments_hint(const GDScriptParser::FunctionNode *p_functio
 			arghint += String::chr(0xFFFF);
 		}
 		const GDScriptParser::ParameterNode *par = p_function->parameters[i];
-		arghint += par->identifier->name.operator String() + ": " + par->get_datatype().to_string();
+
+		if (par->datatype_specifier == nullptr) {
```

Comment:
This looks like it doesn't consider inferred static types (`:=`).

Using `is_hard_type()` might be a better idea.

Reply:
Thanks for your guidance, I have fixed it